specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/apimpolicyfragments.json

```diff
@@ -0,0 +1,343 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApiManagementClient",
+    "description": "Use this REST API for performing operations on policy fragments in Azure API Management deployment.",
+    "version": "2021-04-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/policyFragments": {
+      "get": {
+        "tags": [
+          "PolicyFragment"
+        ],
+        "operationId": "PolicyFragment_ListByService",
+        "description": "Lists all the policy fragment definitions of the Api Management service.",
+        "x-ms-examples": {
+          "ApiManagementListPolicyFragments": {
+            "$ref": "./examples/ApiManagementListPolicyFragments.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns an array of references to policy fragments.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/PolicyFragmentCollection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
```

Comment:
Does apimangement.json need to declare its own ErrorResponse type or can it use the common one from https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json
?